assets/js/aplus.js

```diff
@@ -665,28 +669,22 @@ $(function() {
  * to the next
  * Note! Translation requires that at least 'hl' query parameter is copied
  */
-(function(window) {
-    window.addEventListener('click', function(event) {
-        if (
-            event.target.nodeName === 'A'
-            && event.target.href
-            && event.target.protocol === window.location.protocol
-            && event.target.host === window.location.host // hostname:port
-            && (event.target.search === '' || event.target.search === '?') // no search
-        ) {
-            event.preventDefault(); // stop default redirect
-            event.target.search = window.location.search; // copy query params
-            if (event.target.getAttribute('target') === '_blank') {
-                // Open a new window or tab.
-                window.open(
-                    event.target.href,
-                    '_blank',
-                    'location,menubar,resizable,scrollbars,status,noopener');
-            } else {
-                window.location.href = event.target.href; // redirect the current window
-            }
-        }
-    }, false);
+(function (window) {
+  // The following regex takes the language values from the LANGUAGES variable in the aplus/settings.py, i.e., en and fi.
+  // Therefore, the values must be adjusted accordingly.
+  const regexp = /\?+.*hl=(en|fi)+/gi;
+  const addQueryParameter = (event) => {
```

Comment:
Pointing out to remember that IE 11 does not support arrow functions. From style point of view, I don't see reason to use one here either. Default version is even a bit shorter.

```javascript
const addQueryParameter = (event) => {
function addQueryParameter(event) {
```

Reply:
I think that we should not support IE at all. Even Microsoft is going to stop supporting that browser. https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-365-apps-say-farewell-to-internet-explorer-11-and/ba-p/1591666

Again,  I think we should state somewhere the browsers we support.